cmd/diff.go

```diff
@@ -257,27 +322,32 @@ func fmtDiff(root, baseLine, candidate *profiler.CallMetrics, metricType tableCo
 	}
 
 	// Convert value to ms
-	rootMs := float64(rootVal.Nanoseconds()) / 1.0e6
-	baseMs := float64(baseVal.Nanoseconds()) / 1.0e6
-	candMs := float64(candVal.Nanoseconds()) / 1.0e6
-	absDelta := math.Abs(baseMs - candMs)
+	rootTime := dp.unit.Convert(rootVal)
+	baseTime := dp.unit.Convert(baseVal)
+	candTime := dp.unit.Convert(candVal)
 
-	if candidate == baseLine || absDelta < threshold {
-		switch df {
+	if candidate == baseLine {
+		switch dp.format {
 		case displayTime:
-			return fmt.Sprintf("%1.2f (--)", candMs)
+			return fmt.Sprintf("%s", dp.unit.Format(candTime))
```

Comment:
Why do we need the `%s` here – could it not just be `return dp.unit.Format(candTime)` (or maybe `.String()`)?